staging/kudo/values.yaml

```diff
@@ -0,0 +1,23 @@
+replicaCount: 1
+
+image:
+  repository: kudobuilder/controller
+  tag: v0.7.4
+  pullPolicy: IfNotPresent
+
+
+crds:
+  forceDeploy: false
+
+resources:
```

Comment:
Similar to the above: do we want to set some default limits in the values and have running with no limits be an opt-in config?